apps/console/src/features/applications/components/settings/sign-on-methods/script-based-flow/template-description.tsx

```diff
@@ -111,6 +112,39 @@ export const TemplateDescription: FunctionComponent<TemplateDescriptionPropsInte
         );
 
     };
+
+    const getParameters = () : string[] => {
+        const params = [];
+        if (isObject(template.parametersDescription)) {
+            Object.entries(template.parametersDescription).map(([param]) => {
+                params.push(param);
+            });
+        }
+        return params;
+    }
+
+    const generatePrerequisite = (prerequisite: string, params: Array<string>): ReactElement => {
+        const sentenceArray = prerequisite.split(" ");
+        const modified = [];
+        let content = ""
+        sentenceArray.map((word, index) => {
+            if (params.includes(word)) {
+                modified.push(<span key={index}>{content} </span>)
+                modified.push(<span key={index}><Code>{word}</Code> </span>)
+                content = ""
+            } else {
+                content = content.concat(word + " ");
+            }
+        });
+        modified.push(<span>{content}</span>)
+        return (
+            <p>
+                <React.Fragment>
+                    {modified.map((element) => element)}
+                </React.Fragment>
```

Comment:
Why do we need to wrap it inside a fragment?

Reply:
Resolved in [0aad4b3](https://github.com/wso2/identity-apps/pull/3777/commits/0aad4b38955f51616d9d2927bdaf9c64a2dbfe54)